core/src/main/scala/pureconfig/error/ConfigReaderFailure.scala

```diff
@@ -72,6 +72,16 @@ sealed abstract class ConfigReaderFailure {
   def path: Option[String]
 
   /**
+   * A human-readable description of the failure.
+   */
+  def description: String
+
+  /**
+   * A human-readable description of the failure, along with its location.
+   */
+  def descriptionWithLocation: String = location.fold(description)(_.description + " " + description)
```

Comment:
I agree. Moved the `descriptionWithLocation` method to the `ConfigReaderException` companion object in 53ede88.